.github/workflows/ci.yml

```diff
@@ -25,9 +25,9 @@ jobs:
           # - goVersion: "1"
           #   GOARCH: "ppc64le"
           #   SKIP_PYTHON_BINDINGS_TESTS: "0"
-          - goVersion: "1.12"
-            GOARCH: "amd64"
-            SKIP_PYTHON_BINDINGS_TESTS: "0"
+          # - goVersion: "1.12"
+          #   GOARCH: "amd64"
+          #   SKIP_PYTHON_BINDINGS_TESTS: "0"
```

Comment:
Build s `1.12` pada, nejaka zavislost, ktoru to instaluje pred testami uz nefunguje s touto verziou:
![image](https://user-images.githubusercontent.com/19371734/164282074-bb7820c8-8bbd-4dfe-ade3-b99b6f008acd.png)
![image](https://user-images.githubusercontent.com/19371734/164282109-a9c6e0bc-7021-4795-96f8-a87470c29e86.png)
https://github.com/keboola/go-jsonnet/runs/6097203637?check_suite_focus=true#step:7:15

------------------ 

Neriesim, pockam kym to fixnu v oficialnom repe, ... tu sa nic nemenilo, pada to aj tam.